crates/tower-cmd/src/run.rs

```diff
@@ -291,44 +327,91 @@ impl OutputSink for StdoutSink {
 struct ChannelSink(tokio::sync::mpsc::UnboundedSender<String>);
 impl OutputSink for ChannelSink {
     fn send_line(&self, line: String) {
-        let _ = self.0.send(line);
+        if let Some((ts, msg)) = line.split_once(": ") {
```

Comment:
Man this really makes me think that we should be pushing the timestamp into a parameter to `send_line` on the `OutputSink` trait, considering we're doing it in two places. Probably want to be able to format the ts differently for different cases...

Could be follow-up.